docs/pages/docs/01-cloud-deployment/aws-deployment.md

```diff
@@ -12,53 +12,145 @@ folder: docs
+architecture and configuration and AWS CloudFormation uses the template to provision the required resources (such as EC2 instances, EBS storages, etc.). The deployed application together with its 
+associated resources is referred to as a `stack`.
+- **Grakn KGMS Amazon Machine Image (AMI)**
+
+Provides a Grakn-equipped image that can be used when launching instances. The AMI is specified when starting an instances and you are free to launch as many instances as need be, 
+combine them with instances using different AMIs and orchestrate them.
+
+To commence deployment, click yellow `Continue to Subscribe` button. Once subscribed, you should see the method configuration choice screen:
+
+![](/images/aws-deployment-methods.png)
+
+which allows to either pick the [AMI](#grakn-ami):
+ 
+![](/images/aws-deployment-ami.png)
+ 
+or the [CloudFormation](#grakn-cloudformation) options.
+
+![](/images/aws-deployment-cloudformation.png).
+
+To proceed please pick the required option and press the `Continue to Launch` button.
+ 
+### <a name="grakn-ami"></a>Grakn AMI 
+The AMI launch screen looks as follows:
+![](/images/aws-deployment-ami-launch.png).
+
+and allows to specify the instance parameters. After having specified the parameters, press the launch button to start a Grakn instance.  
+
+The AMI Provides an image with a preinstalled Grakn KGMS. The following list summarises the important locations:
+
+- Grakn dist: `/opt/grakn/`
+- Grakn config: /`opt/grakn/conf/grakn.properties`
+- logs: `/var/log/grakn/`
+
+
+#### Important: storage configuration
+By default Grakn is expecting the storage directory to be `/mnt/data1/`. We recommend attaching an EBS drive to the Grakn instance and mounting it so that storage is located on the EBS drive.
+The instructions how to attach the EBS drive can be found here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-attaching-volume.html
+
+In order to mount the drive, please follow the following procedure:
+* identify the EBS block name and path, to do this run `lsblk` command and find the name of attached EBS device. The path will then be `/dev/<BLOCK_NAME>`.
+* if the drive is not formatted, format it by executing:
+```
+mkfs -t ext4 <BLOCK_HOME>
+```
+
+* mount the drive by executing:
+```
+mount <BLOCK_PATH> /mnt/data1/
+```
+
+#### Running
+Grakn is configured as a service and by default it is not running. Once you have configured the storage, to start grakn the following command needs to be executed as root on target machine:
+```
+systemctl grakn start
+```
+
+To stop run:
+
+```
+systemctl grakn stop
+```
+
+
+### <a name="grakn-cloudformation"></a>CloudFormation
+After initiating launch, we arrive at the CloudFormation stack creation page:
+
+![](/images/aws-cloudformation.png).
+
+To proceed the deployment of Grakn, simple press the `Next` button and you will be taken to the stack parameter page;
```

Comment:
To proceed, simply press ...

This is what we want to say unless pressing next actually deploys Grakn